session.go

```diff
@@ -2190,6 +2204,22 @@ func (s *Session) SetPoolTimeout(timeout time.Duration) {
 	s.m.Unlock()
 }
 
+// SetMaxStalenessSeconds set the maximum of seconds of replication lag from secondaries
+//
+// Relevant documentation:
+//
+//   https://docs.mongodb.com/manual/core/read-preference/#maxstalenessseconds
+//
+func (s *Session) SetMaxStalenessSeconds(seconds int) {
+	s.m.Lock()
```

Comment:
It's usually nicer for users if a function returns an error rather than panicking - it's easier to handle and a little less surprising (it's documented as part of the API that there can be an error, rather than just calling something and it dying).